tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -7,11 +7,12 @@ github source.
 
 The TensorFlow Python API supports Python 2.7 and Python 3.3+.
 
-The GPU version (Linux only) works best with Cuda Toolkit 7.5 and
-cuDNN v4.  other versions are supported (Cuda toolkit >= 7.0 and
-cuDNN 6.5(v2), 7.0(v3), v5) only when installing from sources.
+The GPU version works best with Cuda Toolkit 7.5 and
+cuDNN v4 on Linux.  Other versions are supported (Cuda toolkit >= 7.0 and
```

Comment:
I'd change this to v5, and remove v2 on the line below, since we recently removed cudnnV2 support.